run_hs_metrics.sh

```diff
@@ -22,7 +22,7 @@ METRICS=$5
 OUTDIR="${ANALYSISDIR}/results/Reports/HsMetrics"
 
 # singularity container used by sarek, provides CollectHsMetrics
-CONTAINER=/lupus/ngi/containers/sarek/nfcore-sarek-2.6.1.img
+CONTAINER=/vulpes/ngi/containers/sarek/nfcore-sarek-2.6.1.img
```

Comment:
This path doesn't seem to exist? I did find `/vulpes/ngi/containers/sarek/nfcore-sarek-2.7.img` though.

Reply:
true